boms/pom.xml

```diff
@@ -1,73 +1,34 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
-                             http://maven.apache.org/xsd/maven-4.0.0.xsd">
+     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+               http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>com.google.cloud</groupId>
-  <artifactId>zero-pom-test</artifactId>
-  <version>0.0.1-SNAPSHOT</version>
+  <groupId>com.google.cloud.tools.opensource</groupId>
+  <artifactId>cloud-tools-opensource-parent</artifactId>
   <packaging>pom</packaging>
-
-  <name>Google Cloud Platform Supported Libraries</name>
-  <description>
-    A compatible set of Google Cloud open source libraries.
-  </description>
-
+  <version>1.0-SNAPSHOT</version>
+  <name>BOM Parent</name>
+  <url>https://github.com/GoogleCloudPlatform/cloud-opensource-java/boms</url>
   <organization>
-    <name>Google LLC</name>
-    <url>https://cloud.google.com</url>
+    <name>Google LLC.</name>
```

Comment:
Sometimes there's a period at the end and sometimes not. I don't know which is correct, but can we be consistent?

Also, are we really an LLC?

Reply:
Yes, Google is really an LLC. Alphabet is an Inc. Ping me internally if you want to know about the period.